back_ends/r/exec.R

```diff
@@ -19,24 +20,29 @@ output_type = commandArgs()[11]
 #  return(image_write(fig, path="/sandbox/image_output", format="jpg", flatten=FALSE))
 #}
 
-exec_jpg <- function(code) {
-  library(ggplot2)
+if (is_checking_code) {
```

Comment:
This looks good, but please make sure you've tested it carefully in the app itself (if you haven't already).